src/actions/Actions.ts

```diff
@@ -177,94 +189,102 @@ export const AsyncActions = {
     },
     sharePost: (post: Post): Thunk => {
         return async (dispatch, getState) => {
-            const isQueueEmtpy = getState().postUploadQueue.length === 0;
-            dispatch(InternalActions.queuePostForUpload(post));
+            const getOrCreateOwnFeed = (): LocalFeed => {
+                const ownFeeds = getState().ownFeeds;
+                if (ownFeeds.length === 0) {
+                    const identity = getState().author.identity!;
+                    const address = Swarm.makeFeedAddressFromPublicIdentity(identity);
+                    const author = getState().author;
+                    const ownFeed: LocalFeed = {
+                        posts: [],
+                        authorImage: author.image,
+                        name: author.name,
+                        url: `bzz-feed:/?user=${address.user}`,
```

Comment:
there should be a constant or helper function to assemble the url

Reply:
fixed